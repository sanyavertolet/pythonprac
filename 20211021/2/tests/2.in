:doubleitem item item*2
doubleitem '/sanya_vertolet'
doubleitem 2
doubleitem ['2','2','8']
:f a b c a+b+c
f 1 2 3
quit '{}of{}'
