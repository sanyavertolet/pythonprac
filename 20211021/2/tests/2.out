/sanya_vertolet/sanya_vertolet
4
['2', '2', '8', '2', '2', '8']
6
'3of7'
